triage/billing_summary_generator.py

```
@@ -210,276 +221,425 @@ def generate_billing_summary(
+         bold=True, txt_color=C_WHITE, fill=C1_HEADER, h="center")
+
+    # Compute weekly buckets
+    weekly: dict = defaultdict(lambda: {"gross": 0.0, "lunch": 0.0, "net": 0.0,
+                                        "projects": set(), "days": set()})
     for rec in sorted_records:
         d = rec["date"]
         mon = d - _dt.timedelta(days=d.weekday())
         fri = mon + _dt.timedelta(days=4)
-        wk = mon
+        wk  = mon
         label = f"{mon.strftime('%b %-d')}–{fri.strftime('%-d')}"
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)